<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>3000</width>
    <height>2000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>3000</width>
    <height>2000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="SlideText">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>20</y>
      <width>1620</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>1080</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
      <stylestrategy>PreferAntialias</stylestrategy>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="SelectedSongListLineEdit">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string>List of Selected Music</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="LyricsLineEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>22</y>
      <width>750</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>750</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Lyrics</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QListWidget" name="SelectedSongList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>500</width>
      <height>400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>400</height>
     </size>
    </property>
   </widget>
   <widget class="QListWidget" name="LyricList">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>70</y>
      <width>750</width>
      <height>1400</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>750</width>
      <height>1400</height>
     </size>
    </property>
   </widget>
   <widget class="QSearchBar" name="SearchBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>480</y>
      <width>500</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>40</height>
     </size>
    </property>
    <property name="inputMask">
     <string/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string comment="Search the title of music"/>
    </property>
   </widget>
   <widget class="QListWidget" name="SongList">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>530</y>
      <width>500</width>
      <height>1200</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>500</width>
      <height>1200</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="FontLabel">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>1120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Font-Style</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFontComboBox" name="FontComboBox">
    <property name="geometry">
     <rect>
      <x>1550</x>
      <y>1120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
   </widget>
   <widget class="QLabel" name="FontSizeLabel">
    <property name="geometry">
     <rect>
      <x>1770</x>
      <y>1120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Font-Size</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="FontSizeLine">
    <property name="geometry">
     <rect>
      <x>1980</x>
      <y>1120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>24</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="MultiplierLabel">
    <property name="geometry">
     <rect>
      <x>2190</x>
      <y>1120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>x</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="MultiplierLine">
    <property name="geometry">
     <rect>
      <x>2240</x>
      <y>1120</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>1</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="BoldButton">
    <property name="geometry">
     <rect>
      <x>2800</x>
      <y>1120</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>B</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ProjectButton">
    <property name="geometry">
     <rect>
      <x>2860</x>
      <y>1120</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Project</string>
    </property>
   </widget>
   <widget class="QListWidget" name="ImageList">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>1170</y>
      <width>1620</width>
      <height>610</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>610</height>
     </size>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
   </widget>
   <widget class="QComboBox" name="LyricTypeComboBox">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>1479</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <item>
     <property name="text">
      <string>Verse</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Pre-Chorus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Chorus</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Bridge</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Others</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="OtherLyricTypeEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>1480</y>
      <width>200</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>200</width>
      <height>40</height>
     </size>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QSpinBox" name="LyricTypeNumberSpinBox">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>1480</y>
      <width>50</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>50</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveLyricUpButton">
    <property name="geometry">
     <rect>
      <x>1220</x>
      <y>1480</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="MoveLyricDownButton">
    <property name="geometry">
     <rect>
      <x>1270</x>
      <y>1480</y>
      <width>40</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTextEdit" name="LyricEdit">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>1530</y>
      <width>750</width>
      <height>250</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>750</width>
      <height>250</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="NameBar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1740</y>
      <width>390</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>390</width>
      <height>40</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>9</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeSongButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>1740</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QPushButton" name="NewSongButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>New Song</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteSongButton">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddSongButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="RemoveSongButton">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="AddLyricButton">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteLyricButton">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeLyricButton">
    <property name="geometry">
     <rect>
      <x>1210</x>
      <y>1790</y>
      <width>100</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>100</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string>Change</string>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>1790</y>
      <width>1620</width>
      <height>40</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>40</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="SlideBackground">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>1340</x>
      <y>20</y>
      <width>1620</width>
      <height>1080</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1620</width>
      <height>1080</height>
     </size>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <zorder>SongList</zorder>
   <zorder>SearchBar</zorder>
   <zorder>SelectedSongList</zorder>
   <zorder>NewSongButton</zorder>
   <zorder>RemoveSongButton</zorder>
   <zorder>DeleteSongButton</zorder>
   <zorder>AddSongButton</zorder>
   <zorder>NameBar</zorder>
   <zorder>ChangeSongButton</zorder>
   <zorder>SelectedSongListLineEdit</zorder>
   <zorder>LyricsLineEdit</zorder>
   <zorder>LyricList</zorder>
   <zorder>AddLyricButton</zorder>
   <zorder>DeleteLyricButton</zorder>
   <zorder>ChangeLyricButton</zorder>
   <zorder>LyricTypeComboBox</zorder>
   <zorder>LyricTypeNumberSpinBox</zorder>
   <zorder>OtherLyricTypeEdit</zorder>
   <zorder>SlideBackground</zorder>
   <zorder>ImageList</zorder>
   <zorder>ProjectButton</zorder>
   <zorder>FontComboBox</zorder>
   <zorder>FontLabel</zorder>
   <zorder>FontSizeLabel</zorder>
   <zorder>FontSizeLine</zorder>
   <zorder>MultiplierLabel</zorder>
   <zorder>MultiplierLine</zorder>
   <zorder>MoveLyricDownButton</zorder>
   <zorder>MoveLyricUpButton</zorder>
   <zorder>BoldButton</zorder>
   <zorder>ErrorLabel</zorder>
   <zorder>LyricEdit</zorder>
   <zorder>SlideText</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>3000</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QSearchBar</class>
   <extends>QLineEdit</extends>
   <header location="global">qsearchbar.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
